metrics/api/v1/model_handlers.go

```diff
@@ -33,14 +33,6 @@ import (
 // when manager.model has not been initialized.
 var errModelNotActivated = errors.New("the model is not activated")
 
-var metricNamesConversion = map[string]string{
```

Comment:
Ah damn. How about keeping the mappings, deprecated, & supporting all metrics in the way this PR does? So check if metric is supported, if not check if there's a mapped metric?